Description: Fix manpage whatis entries
Author: Oliver Reiche <oliver.reiche@gmail.com>
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: b/share/man/just-graph-file.5.md
===================================================================
--- a/share/man/just-graph-file.5.md
+++ b/share/man/just-graph-file.5.md
@@ -3,7 +3,7 @@
 NAME
 ====
 
-just graph file - The format of the action graph used by **`just`**(1)
+just-graph-file - The format of the action graph used by **`just`**(1)
 
 DESCRIPTION
 ===========
Index: b/share/man/just-mr-repository-config.5.md
===================================================================
--- a/share/man/just-mr-repository-config.5.md
+++ b/share/man/just-mr-repository-config.5.md
@@ -3,7 +3,7 @@
 NAME
 ====
 
-just-mr repository config - The format of the repository config used by
+just-mr-repository-config - The format of the repository config used by
 **`just-mr`**(1)
 
 DESCRIPTION
Index: b/share/man/just-mrrc.5.md
===================================================================
--- a/share/man/just-mrrc.5.md
+++ b/share/man/just-mrrc.5.md
@@ -3,8 +3,7 @@
 NAME
 ====
 
-just-mr configuration - The format of the configuration used by
-**`just-mr`**(1)
+just-mrrc - The format of the configuration used by **`just-mr`**(1)
 
 DESCRIPTION
 ===========
Index: b/share/man/just-repository-config.5.md
===================================================================
--- a/share/man/just-repository-config.5.md
+++ b/share/man/just-repository-config.5.md
@@ -3,7 +3,7 @@
 NAME
 ====
 
-just repository config - The format of the repository config used by
+just-repository-config - The format of the repository config used by
 **`just`**(1)
 
 DESCRIPTION
Index: b/share/man/just-serve-config.5.md
===================================================================
--- a/share/man/just-serve-config.5.md
+++ b/share/man/just-serve-config.5.md
@@ -3,8 +3,8 @@
 NAME
 ====
 
-**`just`** **`serve`** configuration - The format of the configuration used by
-the **`serve`** subcommand of **`just`**(1)
+just-serve-config - The format of the configuration used by the **`serve`**
+subcommand of **`just`**(1)
 
 DESCRIPTION
 ===========
